Source/SpectrumAnalysis/KTChannelAggregator.cc

```diff
@@ -75,104 +101,199 @@ namespace Katydid
+                    {
+                        if (lineContent.find('#')!=std::string::npos) continue;
+                        std::string token;
+                        std::stringstream ss(lineContent);
+                        int wordCount=0;
+                        while (ss >> token)
+                        {
+                            if(wordCount==0)gridLocationX=std::stod(token);
+                            else if(wordCount==1)gridLocationY=std::stod(token);
+                            else
+                            {
+                                KTDEBUG(agglog,"`grid-text-file` cannot have more than 2 columns"); 
+                                return -1;
+                            }
+                            ++wordCount;
+                        }
+                        // Check to make sure that the grid point is within the active detector volume, skip otherwise
+                        if((pow(gridLocationX,2)+pow(gridLocationY,2))>pow(fActiveRadius,2)) continue;
+                        newAggFreqData.SetNComponents(nTotalGridPoints+1);
+                        newAggFreqData.SetGridPoint(nTotalGridPoints, gridLocationX, gridLocationY,iRing);
+                        ++nTotalGridPoints;
+                    }
+                }
+            }
+            else
+            {
+                // Loop over the grid points and fill the values
+                for (int iGridX = 0; iGridX < fNGrid; ++iGridX)
+                {
+                    double gridLocationX = 0;
+                    GetGridLocation(iGridX, fNGrid, gridLocationX);
+                    for (int iGridY = 0; iGridY < fNGrid; ++iGridY)
+                    {
+                        double gridLocationY = 0;
+                        GetGridLocation(iGridY, fNGrid, gridLocationY);
+                        // Check to make sure that the grid point is within the active detector volume, skip otherwise
+                        if((pow(gridLocationX,2)+pow(gridLocationY,2))>pow(fActiveRadius,2)) continue;
+                        newAggFreqData.SetNComponents(nTotalGridPoints+1);
+                        newAggFreqData.SetGridPoint(nTotalGridPoints, gridLocationX, gridLocationY,iRing);
+                        ++nTotalGridPoints;
+                    }
+                }
+            }
+        }
+        return nTotalGridPoints;
+    }
+
+    bool KTChannelAggregator::PerformPhaseSummation(KTFrequencySpectrumDataFFTWCore& fftwData,KTAggregatedFrequencySpectrumDataFFTW &newAggFreqData)
     {
         const KTFrequencySpectrumFFTW* freqSpectrum = fftwData.GetSpectrumFFTW(0);
         int nTimeBins = freqSpectrum->GetNTimeBins();
         // Get the number of frequency bins from the first component of fftwData
         int nFreqBins = freqSpectrum->GetNFrequencyBins();
-        int nComponents = fftwData.GetNComponents(); // Get number of components
+        int nTotalComponents = fftwData.GetNComponents(); // Get number of components
```

Comment:
`int` --> `unsigned` and more spaces.
Either `if (true)` or `if( true )`, but be consistent.